notebooks/energy_volume_curve.ipynb

```diff
@@ -30,14 +30,28 @@
     "ExecuteTime": {
```

Comment:
<p>Nit: Explain why we're changing the input, i.e. to make it run faster</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Mostly I meant the npoints, which I assume are being changed to make things run a bit faster for this demo and we should be explicit about that.</p><p> </p><p>but now this is a good point, why <em>are </em>you changing the volume range? Isn’t the default 0.1 good enough? It might make one of the fit types perform less well later, but I don’t really see a reason to change it. It feels a bit weird if we’re changing a default sans necessity</p>

Reply:
Ah, I thought the intention was to show how to change those parameters, because after all, it's a tutorial, so it should cover various scenarios. There was already the phrase "For this tutorial, we change the input parameters as follows", so I extended it by "but in principle you can run the simulation without touching the input", which I hope tells there's no need to change those parameters if the user doesn't want to.

Reply:
Ah, that's a good idea too. Why don't we do both: "You can change these defaults in the same way as setting new values in a dictionary. In this case, let's decrease the number of volumes we run calculations for to speed the demo up:" (then reset only num points)